### PR TITLE
Prefix newly added globals in Python with underscores

### PR DESCRIPTION
**Pull request by heinrich5991**
Followup of #1599.

### CODE REVIEW

File: uniffi_bindgen/src/bindings/python/templates/wrapper.py [hunk 1]
```diff
@@ -75,9 +75,6 @@
     {%- for c in ci.callback_interface_definitions() %}
     "{{ c.name()|class_name }}",
     {%- endfor %}
-    {%- if ci.has_async_fns() %}
```
Comment (mhammond) [inline uniffi_bindgen/src/bindings/python/templates/wrapper.py:78]: I don't think we want this change - this function is documented as being exported by the Python bindings.

**Review by mhammond — CHANGES_REQUESTED** [review]
Looks fine except for ` uniffi_set_event_loop` (and maybe I just misunderstood!)

File: uniffi_bindgen/src/bindings/python/templates/wrapper.py (hunk 1 above)
Comment (heinrich5991) [reply uniffi_bindgen/src/bindings/python/templates/wrapper.py:78]: It's still exported. `__all__` just determines what you get when you do `from library import *`. You can still get it using `import library; library.uniffi_set_event_loop` or `from library import uniffi_set_event_loop`.

Comment (mhammond) [reply uniffi_bindgen/src/bindings/python/templates/wrapper.py:78]: I understand that - but I'm not sure this change will meet the expectations of people reading our docs.

Comment (mhammond) [reply uniffi_bindgen/src/bindings/python/templates/wrapper.py:78]: (to be clear, I'm not saying this change is wrong, I'm just not sure it's correct. @badboy @bendk do either of you have thoughts)

Comment (heinrich5991) [reply uniffi_bindgen/src/bindings/python/templates/wrapper.py:78]: Added such a note.

**Review by mhammond — APPROVED** [review]
Thanks!

**Pull request merged**